ibis/backends/tests/test_generic.py

```diff
@@ -1430,6 +1458,7 @@ def test_try_cast_func(con, from_val, to_type, func):
                     raises=ImpalaHiveServer2Error,
                     reason="impala doesn't support OFFSET without ORDER BY",
                 ),
+                pytest.mark.notyet(["mssql"], raises=PyODBCProgrammingError),
```

Comment:
Newly failing, not sure why.

Reply:
This one doesn't look like a new failure -- it's already marked on line 1426

Reply:
Don't need the redundant marker, deleting.

Good catch.